src/h5web/providers/Provider.tsx

```diff
@@ -12,32 +14,29 @@ function Provider(props: Props): ReactElement {
   const { api, children } = props;
 
   // Wait until metadata is fetched before rendering app
-  const { value: metadata, error } = useAsync(async () => api.getMetadata(), [
-    api,
-  ]);
+  const { value: metadata, error } = useAsync(async () => {
+    return api.fetchMetadata();
+  }, [api]);
+
+  const valuesStore = useMemo(
+    () => createFetchStore(api.fetchValue.bind(api)),
+    [api]
+  );
```

Comment:
Creating the values store here, with the provider API's `fetchValue` async method. The store has an internal cache that matches the signature of `fetchValue` (i.e. an object with `HDF5Id` keys and `HDF5Value` values).